charts/ingress-nginx/values.yaml

```diff
@@ -651,6 +651,7 @@ controller:
     labels: {}
     # -- Use an existing PSP instead of creating one
     existingPsp: ""
+    networkPolicyEnabled: false
```

Comment:
I'd rather have this as `controller.admissionWebhooks.patch.networkPolicy.enabled`, so `controller.admissionWebhooks.patch.networkPolicy` is an object inside `controller.admissionWebhooks.patch` as we are already having it for the other network policies and can be extended by other properties.

Reply:
`controller.admissionWebhooks.patch.networkPolicy.enabled` seems to make more sense. So I modified it like that.

Reply:
And tested it.

![image](https://github.com/kubernetes/ingress-nginx/assets/2242405/9d370996-fa82-418a-aea7-d7ac3c20cc0c)